scico/linop/radon_svmbir.py

```diff
@@ -122,23 +150,39 @@ def __init__(self, *args, **kwargs):
 
         self.has_prox = True
 
-    def prox(self, v: JaxArray, lam: float) -> JaxArray:
+        self.max_iterations = max_iterations
+        self.stop_threshold = stop_threshold
+
+    def prox(self, v: JaxArray, lam: float, **kwargs) -> JaxArray:
         v = v.reshape(self.A.svmbir_input_shape)
         y = self.y.reshape(self.A.svmbir_output_shape)
         weights = self.W.diagonal.reshape(self.A.svmbir_output_shape)
         sigma_p = snp.sqrt(lam)
+        if "v0" in kwargs and kwargs["v0"] is not None:
+            v0 = np.reshape(np.array(kwargs["v0"]), self.A.svmbir_input_shape)
```

Comment:
Need to include case with `v0` provided in tests.

Reply:
I added a test for `v0`. (Checks if true prox is same as approximate prox with a good initialization).